### PR TITLE
Add 3rd section, use new layout defined by width

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Also:
* Fixes heading underlines
* Fixes layout width

### CODE REVIEW

File: frontend/components/SignUp/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const SignUp = () => (
+  <Section>
+    <div styleName="copy">
+      <div styleName="adventure">
+        <Heading color="red" weight="bold" underline>
+          A Aventura
+        </Heading>
+      </div>
+
```
Comment (luistorres) [inline frontend/components/SignUp/index.js]: No empty lines in JSX. Please

File: frontend/pages/Home/index.css [hunk 2]
```diff
@@ -1,6 +1,15 @@
 @import '../../styles/reset.css';
 @import '../../styles/layout.css';
+@import '../../styles/breakpoints.css';
 
 .root {
   min-width: $min-content-width;
 }
+
+.root > * {
+  padding: $spacing-168 0 $spacing-168 0;
```
Comment (luistorres) [inline frontend/pages/Home/index.css]: You can write just `padding: $spacing-168 0`

Comment (luistorres) [reply frontend/pages/Home/index.css]: same below

**Review by luistorres — APPROVED** [review]
🚀 🤶

File: frontend/pages/Home/index.css (hunk 2 above)
Comment (jfranciscosousa) [reply frontend/pages/Home/index.css]: Thanks doctor CSS

File: frontend/components/SignUp/index.js (hunk 1 above)
Comment (jfranciscosousa) [reply frontend/components/SignUp/index.js]: Yes sorry

**Pull request merged**